mongoOperator/helpers/MongoResources.py

```diff
@@ -139,3 +91,15 @@ def _createReplicaConfig(cls, cluster_object: V1MongoClusterConfiguration) -> Di
             "version": 1,
             "members": [{"_id": i, "host": cls.getMemberHostname(i, name, namespace)} for i in range(replicas)],
         }
+
+    @classmethod
+    def getConnectionSeeds(cls, cluster_object: V1MongoClusterConfiguration) -> List[str]:
```

Comment:
This method name is a bit weird for what it returns. Maybe we can call it `getMemberHostnames`.